src/styles/utils/variables.scss

```diff
@@ -0,0 +1,3 @@
+$tablet-min-width: 576px;
```

Comment:
Consider adding a variable for mobile breakpoint as well, for better responsiveness.